frontend/src/components/teacher/dashboard/ClassroomsSection.tsx

```diff
@@ -0,0 +1,71 @@
+import type { ReactElement } from "react";
+import React from "react";
+import { Box, Center, Grid } from "@chakra-ui/react";
+
+import ErrorState from "../../common/info/ErrorState";
+import LoadingState from "../../common/info/LoadingState";
+import EmptyClassroomsGoToPageMessage from "../../common/info/messages/EmptyClassroomsGoToPage";
+import ClassroomCard from "../student-management/classroom-summary/ClassroomCard";
+import type { QueryOptions } from "../student-management/classroom-summary/useClassDataQuery";
+import useClassDataQuery from "../student-management/classroom-summary/useClassDataQuery";
+
+const QUERY_DATA_OPTIONS: QueryOptions = {
+  limit: 6,
+  sort: {
+    updatedAt: "DESC",
+  },
+  excludeArchived: true,
+};
+
+const ClassroomsSection = (): ReactElement => {
+  const { loading, error, data } = useClassDataQuery(QUERY_DATA_OPTIONS);
+
+  return (
+    <Box flex="1" w="100%">
+      {loading && (
+        <Center flex="1" margin="15%" mt="calc(15% - 2rem)">
+          <LoadingState />
+        </Center>
+      )}
+      {error && (
+        <Box height="100%" mt={2}>
+          <ErrorState />
+        </Box>
```

Comment:
we should prob standardize the styling for load / error states with theme. 

@joyce-shi do you think its in scope for MVP to do? we can create ticket for it or merge with https://www.notion.so/uwblueprintexecs/Add-loading-state-to-login-pages-and-logout-button-29db3f493a4d412084eddd60af59a107?pvs=4 if its not too much work

Reply:
yeah let’s merge it with one of the existing load tickets. not _super critical_ but definitely very nice to have I think

Reply:
plus we should try and get this merged soon :D I think it’s been sitting for a while :-/

Reply:
indeed it has been merged! added to the linked ticket but ff to move